_opensearch/point-in-time.md

```diff
@@ -0,0 +1,154 @@
+
+## Paginating search results
+
+Besides PIT, there are three ways to [paginate search results]({{site.url}}{{site.baseurl}}/opensearch/search/paginate) in OpenSearch: using the Scroll API, specifying `from` and `size` parameters for your search, and using the `search_after` functionality. However, all three have limitations:
+
+- The Scroll API's search results are frozen at the moment of the request, but they are bound to a particular query. Additionally, scroll can only move forward in the search, so if a request for a page fails, the subsequent request skips that page and returns the following one.
+- If you specify the `from` and `size` parameters for your search, the search results are not frozen in time, so they may be inconsistent because of documents being indexed or deleted. The `from` and `size` feature is not recommended for deep pagination because every page request requires processing of all results and filtering them for the requested page.
+- The `search_after` search results are not frozen in time, so they may be inconsistent because of concurrent indexing or deleting documents.
+
+PIT does not have the limitations of other pagination methods, because PIT search is not bound to a query, and it supports consistent pagination going forward and backward. If you have looked at page one of your results and are now on page two, you will see the same page one if you go back.
+
+## Point in Time (PIT) search
+
+PIT search has the same capabilities as regular search, except PIT search acts on an older dataset, while a regular search acts on a live dataset. PIT search is not bound to a query, so you can run different queries on the same dataset that is frozen in time. 
+
+You can use the [Create PIT API]({{site.url}}{{site.baseurl}}/opensearch/point-in-time) to create a PIT. When you create a PIT for a set of indexes, OpenSearch locks a set of segments for these indexes, freezing them in time. On a lower level, none of the resources required for this PIT are modified or deleted. If the segments that are part of a PIT are merged, OpenSearch retains a copy of these segments for the period of time specified at PIT creation by the `keep_alive` parameter. 
+
+The create PIT operation returns a PIT ID, which you can use to run multiple queries on the frozen dataset. Even though the indexes continue to ingest data and modify or delete documents, the PIT references the data that has not changed since the PIT creation. When your query contains a PIT ID, you don't need to pass the indexes to the search because it will use that PIT. A search with a PIT ID will produce exactly the same result when you run it multiple times.
+
+## Pagination with PIT and search_after
+
+When you run a query with a PIT ID, you can use the `search_after` parameter to retrieve the next page of hits. This gives you control over the order of results in the pages of results.
+
+First, run a search query with a PIT ID:
```

Comment:
As there are no items past this one should we just say "Run a search query with a PIT ID:" ?